src/data/articles/articleContents.json

```diff
@@ -38,6 +38,18 @@
           "hideInOverview": true
         }
       ]
+    },
+    {
+      "name": "Islands",
+      "folderName": "islans",
```

Comment:
```suggestion
      "folderName": "islands",
```

Reply:
Please don't tell me you made this typo on purpose just to make me try the suggestion feature :suspect: 😅

Reply:
nice idea, but i didn't 🤣